05-specifying-sample-designs.Rmd

```diff
@@ -1 +1,705 @@
+
+$$ \begin{array}{rrrr} +1 &+1 &+1 &+1\\ +1&-1&+1&-1\\ +1&+1&-1&-1\\ +1 &-1&-1&+1  \end{array} $$
+The columns specify the strata and the rows the replicate. In the first replicate, all the values are +1 so in each of the stratum, the first PSU would be used in the estimate. In the second replicate, the first PSU would be used in stratum 1 and 3 while the second PSU would be used in stratum 2 and 4. In the third replicate the first PSU would be used in stratum 1 and 2 while the second PSU would be used in stratum 3 and 4. Finally, in the third replicate, the first PSU would be used in stratum 1 and 4 while the second PSU would be used in stratum 2 and 3.
+
+#### The math {-}
+
+A weighted estimate for the full sample is calculated as $\hat{\theta}$ and then a weighted estimate for each replicate is calculated as $\hat{\theta}_r$ for $R$ replicates. Then the standard error of the estimate is calculated as:
+
+$$se(\hat{\theta})=\sqrt{\frac{1}{R} \sum_{r=1}^R \left( \hat{\theta}_r-\hat{\theta}\right)^2}$$
+Specifying replicate weights in R requires specifying the type of replicate weights, the main weight, the replicate weights, and some other options. One of the key options is for `mse`. If `mse=TRUE`, variances are computed around the point estimate whereas if `mse=FALSE`, variances are computed around the mean of the replicates which looks like this:
+
+$$se(\hat{\theta})=\sqrt{\frac{1}{R} \sum_{r=1}^R \left( \hat{\theta}_r-\bar{\theta}\right)^2}$$ where $$\bar{\theta}=\frac{1}{R}\sum_{r=1}^R \hat{\theta}_r$$
+
+The default option for `mse` is to use the global option of "survey.replicates.mse" which is set to `FALSE` initially unless a user changes it. Unless documentation states otherwise, for BRR, set `mse` to `TRUE`.
+
+#### The syntax {-} 
+
+Replicate weights generally come in groups and are sequentially numbered such as PWGTP1, PWGTP2, ..., PWGTP80 in the American Community Survey (ACS) or BRRWT1, BRRWT2, ..., BRRWT96 in the 2015 Residential Energy Consumption Survey (RECS). The {srvyr} package relies on tidy selection^[dplyr documentation on tidy-select: https://dplyr.tidyverse.org/reference/dplyr_tidy_select.html] to choose variables. If replicate weight variables need to be specified with a character vector, use the `all_of` function to select variables from a character vector. Some other methods are also illustrated below in the examples but these apply to any type of replicate weights and not just BRR.
+
+If a dataset had WT0 for the main weight and had 20 BRR weights indicated WT1, WT2, ..., WT20, use the following syntax:
```

Comment:
Below you explicitly state that "both are valid".  You may want to do that here too.